docs/git-commands.md

```diff
@@ -0,0 +1,101 @@
+# Git Hub Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo. This copy of the repo will have the remote name of `origin`.
```

Comment:
Maybe add 'to your local machine' after 'will download a copy of the repo'

Reply:
clone a copy ?